packages/teleport/src/components/NodeList/NodeList.tsx

```diff
@@ -48,7 +48,7 @@ function NodeList(props: NodeListProps) {
 
     const columnKey = Object.getOwnPropertyNames(sortDir)[0];
     const sorted = sortBy(filtered, columnKey);
-    if (sortDir[columnKey] === SortTypes.ASC) {
+    if (sortDir[columnKey] !== SortTypes.ASC) {
```

Comment:
Lets just change the sorting order of the hostname column without changing the default sorting in the Table component because that would impact everything else that uses that table. Just change 

`hostname: SortTypes.ASC` to `hostname: SortTypes.DESC`

```tsx
const [sortDir, setSortDir] = React.useState<Record<string, string>>({
    hostname: SortTypes.DESC,
  });
```